src/app/containers/FauxHeadline/index.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useContext } from 'react';
+import { Headline } from '@bbc/psammead-headings';
+import { textDefaultPropTypes } from '#models/propTypes';
+import { ServiceContext } from '#contexts/ServiceContext';
+import { headlineModelPropTypes } from '#models/propTypes/headline';
+import Fragment from '../Fragment';
+import Blocks from '../Blocks';
+import { GridItemConstrainedLarge } from '#lib/styledGrid';
+
+// missing CSS 'display: block;' in psammead branch `explicitly-set-h1-styles-display-and-font-weight`
+const FauxHeadline = props => <Headline as="strong" {...props} />;
+
+const FauxHeadlineContainer = ({ blocks }) => {
+  const { script, service } = useContext(ServiceContext);
+
+  const arrayOfFragments = blocks[0].model.blocks[0].model.blocks;
+
+  if (!arrayOfFragments || !Array.isArray(arrayOfFragments)) {
+    return null;
+  }
+  const componentsToRender = { fragment: Fragment };
+
+  const renderText = () => (
+    <Blocks blocks={arrayOfFragments} componentsToRender={componentsToRender} />
+  );
+
+  return (
+    <GridItemConstrainedLarge>
+      <FauxHeadline script={script} service={service} aria-hidden="true">
```

Comment:
Seems it could be generalised to a HoC capable of adding the `aria-hidden` tag to anything

Reply:
Yeah could be extended in the future if needed. Currently this logic is seen as an exception for the MAP Headline.